### ISSUE TITLE
action-recognition-0001 action list link does not specify/contain the actions output by the model.

### ISSUE THREAD

**Issue by alfawzaan** [openvinotoolkit/open_model_zoo#1930]
The action-recognition-0001 decoder referenced a link that does not list, specify or contain the action that the model produces as logits. Also if attended to, please kindly list them in the order in which the model output its logits. You can access the link to the model documentation [here](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/intel/action-recognition-0001/description/action-recognition-0001.md) to get a view of what I am saying.

**Comment by vladimir-dudnik** [comment]
@alfawzaan have you noticed in model description reference to kinetic-400 dataset for the list of actions, model was trained to detect?
`Please refer to the kinetics dataset specification to see list of action that are recognised by this composite model.`

**Comment by alfawzaan** [comment]
No, please kindly provide me with the link

**Comment by vladimir-dudnik** [comment]
The link to [Kinetic-400](https://storage.googleapis.com/deepmind-media/Datasets/kinetics400.tar.gz) dataset is the only link we have.

**Comment by alfawzaan** [comment]
wow, we will need to have the list of the first 400 actions so that we can actually the action that was predicted.

**Comment by alfawzaan** [comment]
Any solution to this issue? Because without the action output list, the model seems to be redundant. I think what might help is if there is someone who has access to previously executed projects, perhaps they might have access to the list and can share with us.

**Comment by vladimir-dudnik** [comment]
@alfawzaan the solution is to look at [Kinetic-400](https://storage.googleapis.com/deepmind-media/Datasets/kinetics400.tar.gz) dataset and extract list of action from there.

**Comment by vladimir-dudnik** [comment]
closing as answered
